sentences/pl/cover_HassOpenCover.yaml

```diff
@@ -11,9 +11,9 @@ intents:
         slots:
           device_class: "garage_door"
       - sentences:
-          - "Rolety <area> (otwórz | rozsłoń)"
-          - "(Otwórz | Rozsłoń) rolety <area>"
-          - "(Otwórz | Rozsłoń) <area> [wszystkie] rolety"
+          - "Rolety <area> (otwórz | odsłoń)"
```

Comment:
archaic forms can still be used, so let's keep matching on it.

Reply:
@balloob No problem I can just add the modern form to the list and leave it as is.
Just to make sure - I was not able to find a modern dictionary that references that word, all examples I was able to find are dated back to XVI - XIX centuries.
What's our take for such cases in the future? We want to leave as much flexibility as possible or rather limit the words so we make the voice recognition a bit easier?

Reply:
Flexibility is preferred here since we are just matching text. In the future, when generate training sentences we will need to be more precise.